.rubocop.yml

```diff
@@ -0,0 +1,14 @@
+AllCops:
```

Comment:
Rather than using and customizing Rubocop, We should be linting with the [ChefStyle](https://github.com/chef/chefstyle) that ships with ChefDK.